namer/core/src/test/scala/io/buoyant/namer/booNamer.scala

```diff
@@ -0,0 +1,10 @@
+package io.buoyant.namer
+
+import com.twitter.finagle.Path
+
+/**
+ * Created by alex on 15-3-2016.
+ */
```

Comment:
Maybe remove this comment from these 4 test config files?  We don't tend to include it in any of our other files.